ccm_web/server/src/app.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from '@nestjs/common'
+import { ConfigModule } from '@nestjs/config'
+
+import { AppController } from './app.controller'
+import { AppService } from './app.service'
+import { LtiModule } from './lti/lti.module'
+
+import { validateConfig } from './config'
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
```

Comment:
I see the ConfigModule in NestJS supports loading of configuration from yaml, I suppose this would be similar to json as well. What are your thoughts on changing it to JSON?

Reply:
I'm not opposed to using a YAML or JSON configuration file. The NestJS documentation does suggest that using environment variables is a best practice, but I'm open. I think this could be done in a separate issue though.